docker/go-control-plane/controlplane/README.md

```diff
@@ -0,0 +1 @@
+Copied from (Envoy internal package)[https://github.com/envoyproxy/go-control-plane/tree/main/internal/example].
```

Comment:
I don't find this README particularly helpful. It should either say something about the server that we are running, which also exposes another service to control the xDS server, or I'm fine with completely removing this for now, and adding something later on.